src/components/TagModal.vue

```diff
@@ -177,7 +177,11 @@ export default {
 				showError(this.t('mail', 'Tag already exists'))
 				return
 			}
-			if (displayName !== null && displayName !== '') {
+			if (displayName.trim() === '') {
+				showError(this.t('mail', 'Tag name cannot be empty'))
+				return
+			}
+			if (displayName !== '') {
```

Comment:
isn't this check superfluous if you're testing the trimmed name for emptyness above?

Reply:
true, i didnt add this one, it was there, but i should have removed it instead of simplifying it.